tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/pages/LoginPage.java

```diff
@@ -22,16 +20,33 @@ public class LoginPage {
     private final static SelenideElement loginInput = $("#pf-login-username-id");
     private final static SelenideElement passwordInput = $("#pf-login-password-id");
     private final static SelenideElement loginButton = $("button[type='submit']");
+    private final static SelenideElement warning = $("p.pf-c-form__helper-text.pf-m-error").$("Login attempt blocked");
+
 
     /**
      * Login to hawtio as given user with given password.
      */
+
     public void login(String username, String password) {
         try {
+            String pass = "gojo";
+            int maxRetries = 7;
+            int retries = 0;
+            while (!warning.isDisplayed() && retries < maxRetries) {
+                Selenide.Wait().until(ExpectedConditions.visibilityOf(loginButton));
+                loginInput.shouldBe(editable).setValue(username);
+                passwordInput.shouldBe(editable).setValue(pass);
+                loginButton.shouldBe(enabled).click();
+                retries++;
+            }
+
+
+            Selenide.Wait().until(ExpectedConditions.visibilityOf(loginButton));
```

Comment:
Please create another method to test the throttling behavior and create a Cucumber scenario for it. 
We should be testing that the throttling time is increasing (2 or 3 attempts is enough).